TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -274,7 +274,7 @@ import PromiseKit
         onFinished: @escaping () -> Void
     ) {
         // Add missing linkedBrokers
-        let localUserIds = self.linkedBrokers.flatMap { $0.linkedLogin.userId }
+        let localUserIds: [String] = self.linkedBrokers.flatMap { $0.linkedLogin.userId }
```

Comment:
Without defining the type, Swift 4 would interpret this as `flatMap` all the way down including the strings.